src/EventHandler/EventInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\EventHandler;
+
+interface EventInterface
+{
+    public function getArray():array;
+
+    public function getClassName():string;
```

Comment:
maybe `getDescription` ? What is it used for?

Reply:
The event and it's corresponding listeners have been stored as map of Event name as string key and Listener method callable as value so the getClassname returns the event name whenever the listener value has to be chosen from map.

Reply:
I think the map only needs to be the event name, not the class name? If each event accepts a callable, then it can be just called (eg by `call_user_func()`)

Reply:
So you mean that the callable is stored in the event class only?

Reply:
No, the callable would be stored in the dispatcher, against the event type. By default, the dispatcher has no listeners registered against any events (but maybe some default ones in future). Then, users can register one or more listeners for each event, providing the event handling logic as they do. Remember that one of the main benefits of events is that is allows for extensibility - we are giving users the ability to attach their own custom functionality.